modules/account-lib/test/unit/coin/algo/utils.ts

```diff
@@ -21,4 +21,69 @@ describe('utils', () => {
     const objEncodedByAlgo = algosdk.encodeObj(note);
     should.deepEqual(objEncoded, objEncodedByAlgo);
   });
+
+  it('should return a mnemonic from a secret key ', () => {
+    const keyPair = new Algo.KeyPair();
+    const keys = keyPair.getKeys();
+    let mn;
+    let mnAlgo;
+    if (keys.prv) {
+      mn = Algo.algoUtils.secretKeyToMnemonic(Buffer.from(keys.prv, 'hex'));
+      mnAlgo = algosdk.secretKeyToMnemonic(Buffer.from(keys.prv, 'hex'));
+    }
+    should.equal(mn, mnAlgo);
+  });
+
+  it('it should return error if the secret key is invalid', () => {
+    const secretKeyInValid = 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa';
```

Comment:
Adding data like aaaaaaaaaa is valid but not 100% correct.
If you do that is better to do an assertion for a regex. To real validate that is an invalid hex
0[xX][0-9a-fA-F]+